torch_np/__init__.py

```diff
@@ -15,3 +15,9 @@
 inf = float("inf")
 nan = float("nan")
 from math import pi  # isort: skip
+
+from numpy import s_  # isort: skip
+from numpy import index_exp  # isort: skip
+
+False_ = asarray(False, bool_)
+True_ = asarray(True, bool_)
```

Comment:
What are these for?

Reply:
Why: They're part of NumPy's own namespace, and I was encouraged to add them just because they are used in some of the tests we port from NumPy, and were trivial (`np.s_` and `np.index_exp` don't use arrays/are pure Python).

What: I kinda have no idea :sweat_smile: